layouts/2021/candidates/list.html

```diff
@@ -0,0 +1,90 @@
+{{ define "main" }}
```

Comment:
@astropuffin Am I doing this right? making a layout to match the directory structure?